demosplan/DemosPlanCoreBundle/Logic/FileResponseGenerator/ZipResponseGenerator.php

```diff
@@ -101,25 +106,45 @@ private function fillZipWithData(ZipStream $zipStream, array $file): void
             $this->handleError($e, self::UNKOWN_ERROR, self::XLSX_GENERIC);
         }
 
-        foreach ($file['attachments'] as $attachmentArray) {
-            /** @var File $attachment */
-            foreach ($attachmentArray as $attachment) {
-                try {
+        foreach ($file['attachments'] as $attachmentsArray) {
+            try {
+                foreach ($attachmentsArray['attachments'] as $attachment) {
                     $this->zipExportService->addFileToZipStream(
                         $attachment->getFilePathWithHash(),
                         $file['zipFileName'].'/'.$attachment->getHash().'_'.$attachment->getFilename(),
                         $zipStream
                     );
-                } catch (FileNotFoundException|FileNotReadableException $e) {
-                    $this->handleError($e, self::FIILE_NOT_FOUND_OR_READABLE);
-                    ++$this->errorCount['attachmentNotAddedCount'];
-                } catch (InvalidDataException $e) {
-                    $this->handleError($e, self::FILE_HASH_INVALID);
-                    ++$this->errorCount['attachmentNotAddedCount'];
-                } catch (Exception $e) {
-                    $this->handleError($e, self::UNKOWN_ERROR);
-                    ++$this->errorCount['attachmentUnkownErrorCount'];
                 }
+                /**
+                 * The originalAttachment can be of type { @see File } if there was a stn attachment already set within
+                 * the exportet entity. Or it is an array in case the { @see AssessmentTablePdfExporter } got invoked
```

Comment:
typo exportet

Reply:
done in [5b12050](https://github.com/demos-europe/demosplan-core/pull/2444/commits/5b120501f9de24a0a2ad3fcc1193a6483f9851bd)

Reply:
`typo` slipped in as well :-) Should be `within the exported entity`

Reply:
done in [45f9819](https://github.com/demos-europe/demosplan-core/pull/2444/commits/45f98190a2614e931ec22813a601e7051689e075)